vars/Debian.yml

```diff
@@ -1,5 +1,5 @@
 ---
-_haproxy_repo: "http://http.debian.net/debian {{ ansible_distribution_release }} main"
+_haproxy_repo: "http://http.debian.net/debian"
```

Comment:
Same here.

Reply:
Sorry I didn't apply the rebase correctly in this branch